snack-shop-bff/src/snacks/snacks.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Controller, Get } from '@nestjs/common';
+
+@Controller('snacks')
+export class SnacksController {
+  @Get()
+  findAll() {
+    const snacks = [
+      {
+        _id: '9034b608-fa95-4ff5-9ed9-9d57cb472588',
+        snackName: 'Tortilla Chips',
+        details: {
+          flavor: 'Cheese',
```

Comment:
Same as above, as we're serving business in the UK, I suggest we use British English also in our domain language
```suggestion
          flavour: 'Cheese',
```